data.sql

```diff
@@ -4,4 +4,13 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Agumon', '2020-02-03', 0, true, 10.23),
        ('Gabumon', '2018-11-15', 2, true, 8),
        ('Pikachu', '2021-01-07', 1, false, 15.04),
-       ('Devimon', '2017-05-12', 5, true, 11);
+       ('Devimon', '2017-05-12', 5, true, 11),
+       ('Charmander', '2020-02-08', 0, false, 11),
+----------------------------insert new values for animals table----------------------------------------
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Plantmon', '2021-11-15', 2, true, 5.7),
+        ('Squirtle', '1993-04-02', 3, false, 12.13),
+        ('Angemon', '2005-06-12', 1, true, 45),
```

Comment:
- [ ] You should insert the correct weight data for Charmander `-11`, Plantmon - `-5.7`, Squirtle - `-12.13`, and Angemon - `-45`. They all should be negative values.

Reply:
thanks for your review, done